python/paddle/fluid/tests/unittests/dygraph_to_static/test_list.py

```diff
@@ -138,11 +129,9 @@ def test_list_pop_in_if(x):
         a.append(x + 1)
         a.append(fluid.layers.fill_constant(shape=[2], value=2, dtype="int64"))
     item1 = a.pop(1)
-    a.pop()
```

Comment:
Better keep code `a.pop()`, because `a` is an empty list after `a.pop()` and it's a test case.

Reply:
Thanks, I will re-add it in next PR.